src/metrics.cpp

```diff
@@ -342,9 +347,9 @@ int printStats(bool isScreen, bool mining)
     }
     std::cout << "           " << _("Next upgrade") << " | " << strUpgradeTime << std::endl;
     std::cout << "            " << _("Connections") << " | " << connections << std::endl;
-    std::cout << "  " << _("Network solution rate") << " | " << netsolps << " Sol/s" << std::endl;
+    std::cout << "  " << _("Network solution rate") << " | " << netsolps << " Sol/s " << strprintf("(~ 2^%.4f Sol/s)", netsolpsLog) << std::endl;
```

Comment:
Let' just replace the base-10 representation with the log-2 one for the general view:
```suggestion
    std::cout << "  " << _("Network solution rate") << " | " << strprintf("~ 2^%.4f Sol/s", netsolpsLog) << std::endl;
```